### PR TITLE
Few more additions.

### PR DESCRIPTION
**Pull request by hemanth**
- .mkdown
- .ron
- .gfm

### CODE REVIEW

**Comment by sindresorhus** [comment]
Source? Where are these used?

**Comment by hemanth** [comment]
github gist handles them all :)

**Comment by sindresorhus** [comment]
Where is it documented? You need to provide some source for this.

**Comment by hemanth** [comment]
[source](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1734-L1744)...now, let's make more modules out of this ;) ?

File: markdown-extensions.json [hunk 1]
```diff
@@ -4,5 +4,8 @@
 	"mdown",
 	"mkdn",
 	"mkd",
-	"mdwn"
+	"mdwn",
+	"mkdown",
+	"ron",
+	"gfm"
```
Comment (sindresorhus) [inline markdown-extensions.json:10]: `gfm` is the source selector, not an extension.

**Comment by sindresorhus** [comment]
> let's make more modules out of this ;) ?

Feel free. I only needed this though.

File: markdown-extensions.json (hunk 1 above)
Comment (hemanth) [inline markdown-extensions.json:10]: Ah, Github Flavoured Markdown...hmm ok.

**Comment by hemanth** [comment]
@sindresorhus I was planning to edit this PR :|

NP! I'm still wondering if other extensions will be useful....

**Comment by sindresorhus** [comment]
>  I was planning to edit this PR :|

You did: https://github.com/sindresorhus/markdown-extensions/commit/f3bc724dc28cf8e5a4a362a4aa7613e7f4e0f3c1

**Comment by hemanth** [comment]
Noticed it now, thanks :+1:

**Pull request closed without merging**